charts/mimir-distributed/INTERNAL.md

```diff
@@ -1,8 +1,36 @@
-# Notes on testing mimir-distributed
+# Developer notes
+
+## Notes on the kube manifest templates and configuration
+
+Why do we generate Kubernetes resource definitions from named templates, and not directly ? That is, why do we have
+```yaml
+{{- if not .Values.skipResourceRendering -}}
+{{- include "mimir.querierDeployment" . -}}
+{{- end -}}
+```
+instead of just writing the Deployment template directly ?
+
+[PR #1182](https://github.com/grafana/helm-charts/pull/1182) commit message, quote:
+
+When Mimir is used as a subchart, the rendering of the annotation "checksum/config" on PODs is wrong. This is important to trigger service restart when the config is changed.
+
+There are two options: config is in Mimir or shared with Mimir in which case it can contain templates and values of the parent chart, making it impossible to evaluate in the subchart. Or the config lives in the parent chart, but then it is not possible to reach it in Mimir as subchart and the checksum is checksum of empty string. See: "few important details" in [Helm subcharts and globals](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/)
+
+This patch makes the manifest templates global named templates and thus able to be rendered in the parent context. It is one step short of merging the mimir-distributed chart with enterprise-metrics.
```

Comment:
```suggestion
```
I would consider removing this line as I think "This patch ..." will be confusing to readers post-merge.

Reply:
Ah, I actually missed that this is quoting the PR description. Perhaps we can use the markdown `>` to indicate that this is a quote instead of the start and end quote sentences?